### DISCUSSION TITLE
Gardening Efforts

### DISCUSSION THREAD

**Discussion by laurelsilva** [eggheadio/egghead-next#646] [Ideas & Research]
This outlines possible gardening efforts that we can tackle: 

## Topic Pages

The following topic pages need curation:  

- [Node.js](https://egghead.io/q/node)
- [Angular](https://egghead.io/q/angular)

Each page needs to contain `Beginner`, `Intermediate`, `Advanced` blocks, and feature courses.

## Home Page

We can consider a “page first” approach to curating section where the title becomes clickable and provides an entry point to the main page—a great example of this the Digital Gardening feature and its [page](https://egghead.io/learn/digital-gardening). 

The **Start Learning Here** section can be broken up into its own feature section. Showcasing the most popular courses and beginner-friendly content. 

**NEW Features.** We can also create new features.  A "Learn to Build Accessible Web Applications" feature section would be interesting. Mix and match these resources or any other resource: 
  - [Accessible Cross-Browser CSS Form Styling course](https://egghead.io/courses/accessible-cross-browser-css-form-styling-7297)
  - [Develop Accessible Web Apps with React (course)](https://egghead.io/courses/develop-accessible-web-apps-with-react)
  - [Designing for Inclusive User Experience (talk)](https://egghead.io/talks/egghead-designing-for-inclusive-user-experience)
  - [Accessibility-flavored React Components make your Design System Delicious (talk)](https://egghead.io/talks/react-accessibility-flavored-react-components-make-your-design-system-delicious)
  - [Championing a11y and Being Authentic with Lindsey Kopacz (podcast)](https://egghead.io/podcasts/championing-a11y-and-being-authentic-with-lindsey-kopacz)

**State Management Feature.** The state management playlist can also have its own feature section, very similar to the React State Management section on the react Page, but a smaller version. Mini feature sections? Ideally, it should point to the [State Management Page](https://egghead.io/learn/react/state-management) once we have a solid version of this page.

**Tailwind Feature.** If we consider Tailwind content important, we can create mini-feature sections showcasing the Masterclass and other Tailwind-related content that's not necessarily courses, such as articles and podcasts.

**Staff Picks and Favorites.** This section is not curated and does not center the content on the learner. I would consider replacing this section with a new Feature section. See the [notion document ](https://www.notion.so/egghead/Curated-Stories-Micro-Paths-0413d11047b4448eb30bd69f668e3e9f) containing "groupings" of possible features. 

**Deep Dive into AWS & Learn Something New**. This section doesn't have a clear goal. It's mostly something that we left from our previous gardening efforts. We should consider breaking these two up into their own features or creating a new feature. 

## Instructor Pages

The following instructor pages need curation:

  - [Kent C. Dodds](https://egghead.io/q/resources-by-kent-c-dodds)
  - [Chris Biscardi](https://egghead.io/q/resources-by-chris-biscardi)
  - [Eve Porcello](https://egghead.io/q/resources-by-eve-porcello)
  - [Tomasz Łakomy](https://egghead.io/q/resources-by-tomasz-lakomy)
  - [Hiroko Nishimura](https://egghead.io/q/resources-by-hiro-nishimura)

The page for [Stephanie Eckles](https://egghead.io/q/resources-by-stephanie-eckles) is a good example of a curated instructor page. It showcases recent courses and projects. Note that we are not limited to just courses. We can also add any other resources that the instructor has created (articles, podcast episodes, talks, and partner products).  

## Analytics-driven curation
Analytics can drive curation efforts. [Ahoy](http://app.egghead.io/admin/blazer) records analytics in our database. We can also consider [GA](https://analytics.google.com/analytics/web/#/report/content-pages/a36512724w249816875p231496770/), and course reviews as inputs. 

**Gardening efforts for the home page need to be done on a weekly basis. This is an ongoing process.**

**Comment by laurelsilva** [comment]
https://github.com/eggheadio/egghead-next/pull/649 creates the Node topic page ✅

**Comment by zacjones93** [comment]
Love this centralized spot for talking about curated stories to build on egghead.

I can pick up curating Kent C. Dodds landing page. There are a few different stories that would work well on his page. Obviously, there is EpicReact and TestingJavaScript but Kent has a great track on starting to contribute to open source that would be awesome to feature

**Comment by zacjones93** [comment]
Another activity that might be appropriate in terms of gardening is the instructor case studies that we have on [howtoegghead](https://howtoegghead.com/instructor/case-studies).

While transferring all the data is needed, thinking about ways that would best highlight instructors and stories would be super great

**Comment by lsminter** [comment]
Here is the PR for the Angular Topic page https://github.com/eggheadio/egghead-next/pull/657

**Comment by laurelsilva** [comment]
I'll be working on the State Management Feature and Feature Page.
